docs/source/custom/package.rst

```diff
@@ -1,17 +1,82 @@
-==========================
-Packaging and Distribution
-==========================
+=================================
+Using and Distributing Extensions
+=================================
 
-Ideally, any custom code is made available as a regular Python package.
+Extensions for :py:mod:`cobald` are regular Python code accessible to the interpreter.
+For specific problems, extensions can be defined directly in a Python configuration file.
+General purpose and reusable code should be made available as a regular Python package.
```

Comment:
Why is it important that it is a *regular* Python package. Are there any other options?

```suggestion
General purpose and reusable code should be made available as a Python package.
```